wpeverest/everest-forms
# Feature - Ajax form submission

### All Submissions:






### Changes proposed in this Pull Request:



This PR is about the ajax form submission backend validation for premium fields.

### How to test the changes in this Pull Request:

1. Create a new form or Edit existing form.
2. Go to the settings tab and find Enable ajax form submission.
3. Check the form submission via ajax on the frontend by submitting the form.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Ajax Form Submission
